Doc/library/sqlite3.rst

```diff
@@ -1201,6 +1233,30 @@ Connection objects
    .. _SQLite limit category: https://www.sqlite.org/c3ref/c_limit_attached.html
 
 
+   .. method:: getconfig(op, /)
+
+      Query a boolean connection configuration option.
```

Comment:
```suggestion
      Return the current value of a connection configuration option.
      Currently, only boolean configuration options are supported.
      In the future, other types of options may be supported.
```

Reply:
I think "Query" is fine, and is more concise :)

Reply:
Hmm, I'm not sure the "We might do this in the future, possibly" wording is useful. Users can't rely on it _maybe_ changing in the future. I think I'd just do something like this:

```suggestion
      Query the value of the connection configuration option *op*.
      Only boolean configuration options are supported.
```